extensions/adobe/experience/adcloud/searchads/transaction.schema.json

```diff
@@ -0,0 +1,67 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/adcloud/searchads/transaction",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Search Advertising Transaction Property",
```

Comment:
Transaction properties are not specific to Search Ads. rename to "Advertising Transaction Properties"